### PR TITLE
Resolve ScrollTrigger Uncaught type error

### PR DESCRIPTION
**Pull request by Priyanshu-bit**
Added the ScrollTrigger properly to use ScrollTrigger Function

### CODE REVIEW

**Comment by Priyanshu-bit** [comment]
Hi, @ravin972 , Fixed the issue, Kindly review it

File: src/index.html
Comment (ravin972) [inline src/index.html:1]: Great 👍.

**Review by ravin972 — APPROVED** [review]
👍

**Comment by ravin972** [comment]
Hello,

I'd like to request assistance with showcasing images within the elements of the ".featured" div. It's important that the images change as the user scrolls, providing a dynamic and engaging experience. Here, please take a look at this example – I'd like to replicate a similar functionality on our website. You can find the reference link here: [Video Link](https://gemoo.com/tools/upload-video/share/576947327649021952?codeId=vz8jVLAzEEEK1&card=576947322729103360&origin=videolinkgenerator). Your insights and assistance in implementing this feature would be invaluable.


### And here is the issue link https://github.com/ravin972/cuberto_clone/issues/3#issue-1966684939

Thank you.

**Pull request merged**